src/app/project/component.ts

```diff
@@ -128,7 +128,11 @@ export class ProjectComponent implements OnInit, OnChanges, OnDestroy {
           case 'role':
             return this.compare(this.getRole(a.id), this.getRole(b.id), isAsc);
           case 'clusters':
-            return this.compare(a.clustersNumber, b.clustersNumber, isAsc);
```

Comment:
Changing this to `a.clustersNumber || 0, b.clustersNumber || 0` should be sufficient if this is just about getting rid of undefined values.

Reply:
yep. fixed